README.md

```diff
@@ -1,64 +1,228 @@
-# GYFT - Get Your Freaking Timetable
+<div id="top"></div>
 
-Gets your timetable from ERP and adds it to your **Google Calendar** or gives you an **ICS file** which you can add in any common calendar application.
+<!-- PROJECT SHIELDS -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links-->
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+
+</div>
+
+<!-- PROJECT LOGO -->
+<br />
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/gyft">
+    <img width="140" alt="image" src="https://user-images.githubusercontent.com/86282911/206632284-cb260f57-c612-4ab5-b92b-2172c341ab23.png">
```

Comment:
I'd suggest commenting out the icon until we have a new one.

Reply:
agreed